src/_gettsim/transfers/arbeitslosengeld_2/regelbedarf.py

```diff
@@ -0,0 +1,509 @@
+        out = (
+            arbeitsl_geld_2_params["regelsatz"]
+            * (arbeitsl_geld_2_params["anteil_regelsatz_erwachsene"]["zwei_erwachsene"])
+        )
+    # This observation is not a child, so BG has 1 adult
+    elif kindersatz_m == 0.0:
+        out = arbeitsl_geld_2_params["regelsatz"]
+    else:
+        out = 0.0
+
+    return out * (1 + mehrbedarf_alleinerziehend_m)
+
+
+@policy_function(start_date="2011-01-01", leaf_name="erwachsenensatz_m")
+def arbeitsl_geld_2_erwachsenensatz_ab_2011_m(
+    mehrbedarf_alleinerziehend_m: float,
+    kindersatz_m: float,
+    demograpics__p_id_einstandspartner: int,
+    arbeitsl_geld_2_params: dict,
+) -> float:
+    """Basic monthly subsistence / SGB II needs for adults without dwelling since 2011.
+
+    Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
+
+    Parameters
+    ----------
+    mehrbedarf_alleinerziehend_m
+        See :func:`mehrbedarf_alleinerziehend_m`.
+    kindersatz_m
+        See :func:`kindersatz_m`.
+    demograpics__p_id_einstandspartner
+        See basic input variable :ref:`demograpics__p_id_einstandspartner`.
+    arbeitsl_geld_2_params
+        See params documentation :ref:`arbeitsl_geld_2_params <arbeitsl_geld_2_params>`.
+
+    Returns
+    -------
+    float with the minimum needs of an household in Euro.
+
+    """
+    # BG has 2 adults
+    if demograpics__p_id_einstandspartner >= 0:
+        out = arbeitsl_geld_2_params["regelsatz"][2]
+    # This observation is not a child, so BG has 1 adult
+    elif kindersatz_m == 0.0:
+        out = arbeitsl_geld_2_params["regelsatz"][1]
+    else:
+        out = 0.0
+
+    return out * (1 + mehrbedarf_alleinerziehend_m)
+
+
+@policy_function()
+def regelsatz_m(
+    erwachsenensatz_m: float,
+    kindersatz_m: float,
+) -> float:
+    """Calculate basic monthly subsistence without dwelling until 2010.
+
+    Parameters
+    ----------
+    erwachsenensatz_m
+        See :func:`erwachsenensatz_m`.
+    kindersatz_m
+        See :func:`kindersatz_m`.
+
+    Returns
+    -------
+
+
+    """
+    return erwachsenensatz_m + kindersatz_m
+
+
+@policy_function(end_date="2022-12-31", leaf_name="kosten_unterkunft_m")
+def kosten_unterkunft_m_bis_2022(
+    berechtigte_wohnfläche: float,
+    warmmiete_je_qm_m: float,
+) -> float:
+    """Calculate costs of living eligible to claim until 2022.
+
+    Note: Since 2023, Arbeitslosengeld 2 is referred to as Bürgergeld.
+    Parameters
+    ----------
+    berechtigte_wohnfläche
+        See :func:`berechtigte_wohnfläche`.
+    warmmiete_je_qm_m
+        See :func:`warmmiete_je_qm_m`.
+
+    Returns
+    -------
+    float with total monthly cost of rent.
+
+    """
+    return berechtigte_wohnfläche * warmmiete_je_qm_m
+
+
+@policy_function(start_date="2023-01-01", leaf_name="kosten_unterkunft_m")
+def kosten_unterkunft_m_ab_2023(
+    bruttokaltmiete_m: float,
+    heizkosten_m: float,
+    in_vorjahr_bezogen: bool,
```

Comment:
```suggestion
    arbeitslosengeld_2_bezug_im_vorjahr: bool,
```
or the like. Too unclear what this refers to otherwise, I think? But maybe leave as is.